src/components/ui/spinner.tsx

```
@@ -0,0 +1,10 @@
+import { cn } from "@/lib/utils"
+import { SpinnerIcon } from "@phosphor-icons/react"
```

ESLint

[error] `@phosphor-icons/react` import should occur before import of `@/lib/utils`

(import/order)